tests/exceptions_work.rs

```diff
@@ -68,6 +85,40 @@ fn test_static_message() {
     assert_eq!("hi".to_owned(), error.into_origin());
 }
 
+
+#[test]
+fn serialize_json() {
+
+    let error = throw_with_context3().unwrap_err();
+    let json = serde_json::to_string(&error).unwrap();
+    assert_eq!("{\"points\":[{\"line\":41,\"column\":5,\"module_path\":\"exceptions_work\",\"file\":\"tests/exceptions_work.rs\"},{\"line\":45,\"column\":5,\"module_path\":\"exceptions_work\",\"file\":\"tests/exceptions_work.rs\"},{\"line\":50,\"column\":5,\"module_path\":\"exceptions_work\",\"file\":\"tests/exceptions_work.rs\"}],\"context\":[{\"key\":\"code\",\"value\":78},{\"key\":\"application\",\"value\":\"rust_core\"},{\"key\":\"project_secret\",\"value\":\"omega\"},{\"key\":\"score\",\"value\":0.75},{\"key\":\"height\",\"value\":948}],\"error\":\"Error with context\"}", json);
```

Comment:
Minor nit: string could be simpler within a raw string literal (`r#" string which contains " freely "#`). Something like
```
r#"{"points":[{"line":41,"column":5,"module_path":"exceptions_work","file":"tests/exceptions_work.rs"},{"line":45,"column":5,"module_path":"exceptions_work","file":"tests/exceptions_work.rs"},{"line":50,"column":5,"module_path":"exceptions_work","file":"tests/exceptions_work.rs"}],"context":[{"key":"code","value":78},{"key":"application","value":"rust_core"},{"key":"project_secret","value":"omega"},{"key":"score","value":0.75},{"key":"height","value":948}],"error":"Error with context"}"#
```